src/Microsoft.IIS.Administration.Certificates/Microsoft.IIS.Administration.Certificates.csproj

```diff
@@ -19,7 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="MicroBuild.Core" Version="$(MicroBuildVersion)" />
-    <PackageReference Include="System.Security.Cryptography.X509Certificates" Version="4.1.0" />
+    <PackageReference Include="System.Security.Cryptography.X509Certificates" Version="4.1.*" />
```

Comment:
Why wildcard?

Reply:
Allowing security patches is fine here. Currently, >= 4.1.2 is being required.